Submitted By:            Ragnar Thomsen <rthomsen_at_linuxfromscratch_dot_org>
Date:                    2012-07-21
Initial Package Version: 0.99.0
Origin:                  OpenSUSE
Description:             Removes the unimplemented "remember authorization" checkbox.

diff -Naur a/AuthDialog.ui b/AuthDialog.ui
--- a/AuthDialog.ui	2010-12-09 00:40:22.000000000 +0100
+++ b/AuthDialog.ui	2012-07-21 19:01:38.532138834 +0200
@@ -114,39 +114,6 @@
      </property>
     </widget>
    </item>
-   <item row="4" column="1" colspan="3" >
-    <widget class="QCheckBox" name="cbRemember" >
-     <property name="text" >
-      <string>Remember authorization</string>
-     </property>
-     <property name="checked" >
-      <bool>true</bool>
-     </property>
-    </widget>
-   </item>
-   <item row="5" column="2" colspan="2" >
-    <widget class="QCheckBox" name="cbSessionOnly" >
-     <property name="text" >
-      <string>For this session only</string>
-     </property>
-    </widget>
-   </item>
-   <item row="5" column="1" >
-    <spacer name="horizontalSpacer" >
-     <property name="orientation" >
-      <enum>Qt::Horizontal</enum>
-     </property>
-     <property name="sizeType" >
-      <enum>QSizePolicy::Fixed</enum>
-     </property>
-     <property name="sizeHint" stdset="0" >
-      <size>
-       <width>15</width>
-       <height>10</height>
-      </size>
-     </property>
-    </spacer>
-   </item>
    <item row="1" column="1" colspan="3" >
     <widget class="KComboBox" name="userCB" />
    </item>
